File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.php
<?php

class Person {
    private $name;
    private $age;

    // Constructor
    public function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }

    // Method
    public function print() {
        echo "Name: {$this->name}, Age: {$this->age}\n";
    }
}

// Standalone function
function add($a, $b) {
    return $a + $b;
}

?>

File: daneel/daneel/services/code_analysis/analysis/test_fixtures/simple.rs
struct Person {
    name: String,
    age: i32,
}

impl Person {
    // Constructor (associated function)
    fn new(name: &str, age: i32) -> Person {
        Person {
            name: String::from(name),
            age,
        }
    }

    // Method
    fn print(&self) {
        println!("Name: {}, Age: {}", self.name, self.age);
    }
}

// Standalone function
fn add(a: i32, b: i32) -> i32 {
    a + b
}
